setup.py

```diff
@@ -71,7 +71,7 @@ def run(self):
 # Dependencies
 ################################################################################
 
-install_requires = []
+install_requires = ['click']
```

Comment:
It's too bad you need this dep even if you're not using the command line utilities :/

Reply:
Yeah, the main reason I use this instead of directly argparse is click has utilities for doing testing.